source/_posts/2017/2017-01-20-4-emotional-reasons-I-quit-my-twitter.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: "4 Emotional and Honest Reasons I quit my Twitter"
+perex: '''
+    Nowadays we tend to live busy life with all those social networks, notifications and pings. So busy, we don't have emotional space to just evaluate "How do I like that?". In my life, this creates cycles, where I keep doing over and over the same thing without any progress. After few months feeling bad with Twitter, I've finally decided to quit. Rather from technical point of view, I will focus on my feelings. Because that's what matters most.
+'''
+lang: en
+---
+
+Before I started this article, I read 
+[I Quit Twitter And My Life Got Better. Here's Why I'm Going Back](http://www.forbes.com/sites/laurenorsini/2015/11/11/i-quit-twitter-and-my-life-got-better-heres-why-im-going-back). In the article Lauren describes what was it like to have week of Twitter and why she got back. If you use Twitter, read it too. There are some great insights on autopilot. 
+
+So, I felt bad using Twitter last few months. I wasn't sure if I really want to quit Twitter, so I tried small experiments: 
+
+- I un-followed people down to zero.
+- I limited my time there to end of the day. 
+- I tried different approach Tweets and retweets.
+- I talked with friend how do they enjoy Twitter.
+- I tried pools to engage with followers in new way.
+- I requested feedback from marketing expert to my Tweets.
+
+I really tried, but nothing of this made me feel significantly better. So last experiment: I quit.
+
+*Interesting. This is very similar to my approach to crisis in relationships.*
+
+What were those deal breakers that tear us apart?
+
+
+## 1. I was Addicted - Give me my Dopamine Shot!
+
+Every like made me happy. Every retweet made happier three-times. I only wanted to invest just a little attention, so I refocused on numbers instead of content.
+It's like trying to build a deep connection with somebody having a 20 minutes a days. This frustrated me a lot, because I prefer working on content over collecting numbers.
+
+I talked about my addiction with [Petr Vácha](http://petrvacha.com/) and my intentions to quit, and he **suggested me to use it only once a day**.
+I tried it for few days and it was great. I got higher dopamine shots in the end of the day.
+
+The only issue for me was **I had to focus to not to focus on Twitter throughout the whole day**. It was like to starve the whole day to eat once in the evening.
+It worked technically, **but it didn't feel right**. I could spend 5 minutes on 10 social networks and spend less then hour, but I want to enjoy this by heart, not by my inner computer. 
+
+I know I can't do something for only 50 %. I either do it close to 100 % or go another way. 
```

Comment:
Maybe there is... this is probably more understandable to people who don't know the idiom :D (us)

Let me know if you find it. Or send PR.